airbyte-oauth/src/main/java/io/airbyte/oauth/google/GoogleAdsOauthFlow.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2020 Airbyte
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.airbyte.oauth.google;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.airbyte.config.persistence.ConfigRepository;
+
+public class GoogleAdsOauthFlow extends GoogleOAuthFlow {
+
+  public GoogleAdsOauthFlow(ConfigRepository configRepository) {
+    super(configRepository, "https://www.googleapis.com/auth/adwords");
```

Comment:
Btw is it required that in GoogleAnalytics the URL is encoded as `https%3A//` for the redirect URL? 

Is there a function to make them well-formatted as redirect URL?

Reply:
I introduce some additional code to handle these in https://github.com/airbytehq/airbyte/issues/5993

Reply:
I introduce some additional code to handle these in https://github.com/airbytehq/airbyte/issues/5993

Reply:
good idea. done